index.html

```diff
@@ -71,7 +69,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
                             <h3>Monero is fungible.</h3>
-                            <p> Monero is fungible because it is private by default. In its current state, it is extremely unlikely that Monero will ever be blacklisted by exchanges or vendors due to its association in previous transactions.</p>
+                            <p> Monero is fungible because it is private by default. This means no one can tell one Monero from another Monero and therefore your funds can't be blacklisted by exchanges or vendors.</p>
```

Comment:
Monero is fungible because it is private by default. Monero will not be blacklisted by exchanges or vendors due to its association in previous transactions.